ui-components/src/helpers/filters.ts

```diff
@@ -80,7 +80,11 @@ export function encodeToFrontendFilterString(filterState: ListingFilterState) {
   for (const filterType in filterState) {
     const value = filterState[filterType]
     if (filterType in FrontendListingFilterStateKeys && value !== undefined && value !== "") {
-      queryString += `&${filterType}=${value}`
+      if (filterType === "includeNulls" && value === false) {
+        queryString += ""
+      } else {
+        queryString += `&${filterType}=${value}`
+      }
```

Comment:
```suggestion
      if (filterType === FrontendListingFilterStateKeys.includeNulls && value === false) {
        continue
      }
      queryString += `&${filterType}=${value}`
```

very nit but this is a little cleaner, and uses the key via a variable instead of the name of the key directly.

Reply:
or yeah do what abby said.